sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3897,6 +3897,7 @@ _ZNK2cl4sycl15interop_handler12GetNativeMemEPNS0_6detail16AccessorImplHostE
 _ZNK2cl4sycl15interop_handler14GetNativeQueueEv
 _ZNK2cl4sycl16default_selectorclERKNS0_6deviceE
 _ZNK2cl4sycl20accelerator_selectorclERKNS0_6deviceE
+_ZNK2cl4sycl5event11get_backendEv
```

Comment:
Should minor/patch version of library be increased according to ABI policy?

Reply:
No, it'll be increased before release is tagged.